EMOBON00092/provenance.json

```diff
@@ -0,0 +1,64 @@
+{ "@context": {
+    "@base":"https://github.com/emo-bon/",
+    "@vocab":"https://w3id.org/ro/crate/1.2-DRAFT/context"
+},
+  "@graph": [
```

Comment:
seems like we are mentioning the input and result here in this first provenance-draft file

but I kind off miss any statements about 
- the 'agent' (who executed this when/where) 
- the circumstance (what software, version)  was used ...

I think prov-o has provisioning for that?